backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/mapper/SubscriptionModelMapper.java

```diff
@@ -4,17 +4,26 @@
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 import org.mapstruct.NullValuePropertyMappingStrategy;
+import org.springframework.beans.factory.annotation.Autowired;
 
+import ca.gov.dtsstn.cdcp.api.service.AlertTypeService;
 import ca.gov.dtsstn.cdcp.api.service.domain.Subscription;
 import ca.gov.dtsstn.cdcp.api.web.v1.model.SubscriptionModel;
 import jakarta.annotation.Nullable;
 
 @Mapper(componentModel = "spring")
-public interface SubscriptionModelMapper {
+public abstract class SubscriptionModelMapper {
+
+	@Autowired
+	protected AlertTypeService alertTypeService;
```

Comment:
This can be removed, and the mapper can be converted to an interface.

Reply:
removed